openshift/template.yaml

```diff
@@ -218,7 +218,7 @@ parameters:
 - name: DEPLOYMENT_TYPE
   value: "SaaS"
 - name: INSTALLER_CACHE_CAPACITY
-  value: "6442450944"
+  value: "32 GiB"
```

Comment:
I would expect this to use the same memory units as kubernetes resources.
ref: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory

Reply:
This can be ignored too if we're sticking with "github.com/alecthomas/units"

Reply:
I think we are sticking with github.com/alecthomas/units for now